test/sequential/test-buffer-bad-overload.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+var common = require('../common');
+var assert = require('assert');
+
```

Comment:
also, you can shorten `var common = require('../common');` to simply `require('../common');` since `common` is not directly used.